### ISSUE TITLE
Cell areas on symmetric grid

### ISSUE THREAD

**Issue by jkrasting** [raphaeldussin/static_downsampler#1]
When using `sum_on_supergrid()` to construct cell areas, `outputgrid="symetric"` does not increase the array size by i+1, j+1.

Example:
```
area = sum_on_supergrid(ds, "area", "Q", outputgrid="symetric")
```

This capability would help address https://github.com/raphaeldussin/om4labs/issues/41

**Comment by raphaeldussin** [comment]
@jkrasting can you see if #2 solves your problem?

**Comment by jkrasting** [comment]
@raphaeldussin - it's producing the right array sizes.  So far looks reasonable.  Once /archive comes back online, I'll test it out more thoroughly with Dora. Thanks for the quick fix.

**Comment by jkrasting** [comment]
The shape of the area array is always (j+1,i+1) making it only consistent for the corner points.  (Results below are for `outputgrid="symmetric"`.)

Array sizes for point_type=T:
geolat shape: (1080, 1440) 
geolon shape: (1080, 1440) 
area shape: (**1081**, **1441**) 

Array sizes for point_type=Q:
geolat shape: (1081, 1441) 
geolon shape: (1081, 1441) 
area shape: (1081, 1441) 

Array sizes for point_type=U:
geolat shape: (1080, 1441) 
geolon shape: (1080, 1441) 
area shape: (**1081**, 1441)
 
Array sizes for point_type=V:
geolat shape: (1081, 1440) 
geolon shape: (1081, 1440) 
area shape: (1081, **1441**)

**Comment by raphaeldussin** [comment]
I anticipated that more changes were gonna be necessary in that function, working on it

**Comment by raphaeldussin** [comment]
closed by #2
